CHANGELOG.md

```diff
@@ -20,5 +20,6 @@ For information on changes in released versions of Teku, see the [releases page]
 - Optimised fork choice to avoid unnecessary copying, reducing CPU and memory usage.
 
 ### Bug Fixes
-- Reduced verbosity of warning message when SIGHUP can't be interecepted (e.g. on Windows)
+- Reduced verbosity of warning message when SIGHUP can't be intercepted (e.g. on Windows)
 - Fixed build failure when checked out as a shallow clone. Shallow clones are still not recommended as the version number cannot be determined correctly.
+- Logs generated during voluntary exit command processing should no longer contain usernames or passwords.
```

Comment:
```suggestion
- Debug logs generated during voluntary exit command processing no longer contain usernames or passwords.
```

It only printed at debug level right?

Reply:
it was on stdout, that's the only place I found it...

Reply:
fair enough, just remove the `should` in that case.